### PR TITLE
Update climate_HassClimateGetTemperature.yaml

### PR DESCRIPTION
**Pull request by diplix**
added some variations

### CODE REVIEW

File: sentences/de/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -6,5 +6,7 @@ intents:
           - "Wie hoch ist die Temperatur [<area>]"
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
-          - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie hoch ist die Temperatur (vom Thermostat | der Heizung) <area>"
```
Comment (mib1185) [inline sentences/de/climate_HassClimateGetTemperature.yaml:9]:
```suggestion
          - "Wie hoch ist die Temperatur ((von dem | vom | des) Thermostat[es] | [von] der Heizung) <area>"
```

would also be great to cover these variations by tests 👍

File: sentences/de/climate_HassClimateGetTemperature.yaml [hunk 2]
```diff
@@ -6,5 +6,7 @@ intents:
...
+          - "Wie (warm | kalt) ist es <area>"
```
Comment (mib1185) [inline sentences/de/climate_HassClimateGetTemperature.yaml:10]: duplicate of #313

File: sentences/de/climate_HassClimateGetTemperature.yaml [hunk 3]
```diff
@@ -6,5 +6,7 @@ intents:
...
+          - "Wie viel Grad (ist es | sind [es]) <area>"
```
Comment (mib1185) [inline sentences/de/climate_HassClimateGetTemperature.yaml:11]:
```suggestion
          - "Wie viel Grad ((ist | hat | sind) [es]) <area>"
```

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @diplix 
thanks for your contribution 👍 
i've left some comments

File: sentences/de/climate_HassClimateGetTemperature.yaml (hunk 1 above)
Comment (diplix) [reply sentences/de/climate_HassClimateGetTemperature.yaml:9]: hu, there are tests. i’ll look into it.

**Review by Scorpoon — APPROVED** [review]
@mib1185 i merged the commits from 3 pr's u've mentioned and fixed the failing test. Already closed the other 3. I guess it's resolved now, if u have no other suggestions xD

I also made a slight change to _common, it will cover now device names like "Heizung Küche", but also "die Heizung (<"name">) in der Küche (<"area">)" (did have to make quotes around the words name and area). I guess its very common to use device names like that and also not everyone might have set the area.
Right now I'm not sure if we should handle that this way.
What do you think?

File: sentences/de/climate_HassClimateGetTemperature.yaml [hunk 4]
```diff
@@ -6,5 +6,6 @@ intents:
           - "Wie hoch ist die Temperatur [<area>]"
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
-          - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie hoch ist die Temperatur ((von dem | vom | des) Thermostat[es] | [von] <name>) <area>"
+          - "Wie (warm | heiß | kalt) ist [es] [<area>]"
+          - "Wie viel Grad ((ist | hat | sind) [es]) <area>"
```
Comment (mib1185) [inline sentences/de/climate_HassClimateGetTemperature.yaml:11]:
```suggestion
          - "Wie viel Grad (ist | hat | sind) [es] <area>"
```

**Review by Scorpoon — APPROVED** [review]
Merged commits from other PR's which where mentioned by @mib1185 and made additional changes to fix the tests. Another PR/Issue will be opened cause of this [comment](https://github.com/home-assistant/intents/pull/445/#pullrequestreview-1241040217)

File: sentences/de/climate_HassClimateGetTemperature.yaml [hunk 5]
```diff
@@ -6,5 +6,6 @@ intents:
           - "Wie hoch ist die Temperatur [<area>]"
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
-          - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie hoch ist die Temperatur ((von dem | vom | des) Thermostat[es] | [von] <name>) <area>"
```
Comment (mib1185) [inline sentences/de/climate_HassClimateGetTemperature.yaml:9]:
```suggestion
          - "Wie hoch ist die Temperatur ((von dem | vom | des) Thermostat[es] | (von | vom) <name>) [<area>]"
```
With this we can skip `Wie hoch ist die Temperatur (von | vom) <name>` (line 8)

File: sentences/de/_common.yaml [hunk 6]
```diff
@@ -48,7 +48,7 @@ lists:
         out: "Fahrenheit"
 expansion_rules:
   name: "[(der | den | die | das)] {name}"
-  area: "(in (der | dem) | im) {area}"
+  area: "[(in (der | dem) | im | der | die | das)] {area}"
```
Comment (mib1185) [inline sentences/de/_common.yaml:51]: tbh, I don't get it, what this brings us 🙈
do you have an example, what is now covered?

**Review by mib1185 — COMMENTED** [review]
hi @Scorpoon
Thanks for merging these PR into one 👍
Nevertheless I have left some comments 🙈

File: sentences/de/_common.yaml (hunk 6 above)
Comment (easterapps) [reply sentences/de/_common.yaml:51]: hi @Scorpoon, could you please answer @diplix's question?

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @diplix 👋 

It seems this PR got a bit stale, and thus collected some merge conflicts over time. Could you please take a look and resolve the merge conflicts?

Thanks 👍 

../Frenck

**Comment by frenck** [comment]
Because there hasn't been any activity on this PR for quite some time now, I've decided to close it for being stale.

Feel free to re-open this PR when you are ready to pick up work on it again 👍

../Frenck

**Pull request closed without merging**